models/article.js

```diff
@@ -54,6 +54,10 @@ const articles = (sequelize, DataTypes) => {
       foreignKey: 'articleSlug',
       sourceKey: 'slug'
     });
+    Article.hasMany(models.ReportedArticle, {
+      foreignKey: 'articleId',
```

Comment:
In the URL you are using articleSlug but in the models, you use to store the articleId. Can't you use only one of them?

Reply:
That's a great suggestion, let us work on it.